### PR TITLE
market creation date time fixes

### LINKED ISSUE #5492: Market creation issues
- lets me try to create market w/o enough rep can click confirm
- reporting start time doesn't let me enter today or tomorrow as dates for some reason

### PR DESCRIPTION
**Pull request by bthaile**
fixed trading form bug for pre-liquidity
fix date selector for max market end time, had to get off contract.
check user has enough balance of DAI and REP before enabling create market. 

fixes https://github.com/AugurProject/augur/issues/5492

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). ✌️
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/7g6hl99**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/create-market/components/review.tsx [hunk 1]
```diff
@@ -276,7 +277,7 @@ export default class Review extends React.Component<
     const noEth = s.insufficientFunds[ETH];
     const noRep = s.insufficientFunds[REP];
     const noDai = s.insufficientFunds[DAI];
-
+    this.props.setEnableCreate(noEth || noRep || noDai);
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/components/review.tsx:280]: shouldn't we check if they have enough to create the market?

seeing this
Screen Shot 2020-01-13 at 9 13 02 AM

**Review by phoebemirman — CHANGES_REQUESTED** [review]
see comment

File: packages/augur-ui/src/modules/create-market/components/review.tsx (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/components/review.tsx:280]: this does check if the user has enough to create the market

augur/packages/augur-ui/src/modules/markets/helpers/insufficient-funds.ts

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**